Fun Commands/Mini-Games/Dyno-RPG/register.txt

```diff
@@ -0,0 +1,11 @@
+{delete}
+{silent}
+{!role +Adventurer}
+{!announce {#the-world-gates} Welcome to the world of **Avelon**, {user.username}! This is a cruel world that the gods have long since forsaken. Darkness and torment lies around every corner, and I can not guarantee that you will maintain your sanity. But rest assured, the **Divine Council** will watch over you. Now, head forth with caution young Adventurer, and claim your place in this forsaken world.}
```

Comment:
Small possible spelling change
```suggestion
{!announce {#the-world-gates} Welcome to the world of **Avelon**, {user.username}! This is a cruel world that the gods have long since forsaken. Darkness and torment lies around every corner, and I cannot guarantee that you will maintain your sanity. But rest assured, the **Divine Council** will watch over you. Now, head forth with caution young Adventurer, and claim your place in this forsaken world.}
```

Reply:
Fixed